### DISCUSSION TITLE
6.0.2 (Better gauges, full G-Mode support)

### DISCUSSION THREAD

**Discussion by T-Troll** [T-Troll/alienfx-tools#141] [Announcements]
### Changes in this release:

All fan-control apps:
- New: Full G-Mode switch support added. Checkbox available in GUI (and keyboard shortcut), 2 new commands - `setgmode` and `gmode` - added into CLI.

All fan-control GUI:
- Bug fixed: Incorrect temperature column size in some situation.

`AlienFX-GUI`:
- New: Gauge now have 5 types, "Diagonal" divided to "Diagonal (left)" - from left to right and "Diagonal (right)" - from right to left.
- New: "Reverse" setting added into gauge setting. It's reversing fill direction - to from right to left or from bottom to top.

<hr /><em>This discussion was created from the release <a href='https://github.com/T-Troll/alienfx-tools/releases/tag/6.0.2'>6.0.2 (Better gauges, full G-Mode support)</a>.</em>

**Comment by Rixtho** [comment]
Since downloading this version I am not able to open `alienfx-gui`. `alienfx-mon` and `alienfan-gui` work though. I already tried the repair function of the installer and uninstalling the tool completely and reinstalling it.

**Reply by T-Troll** [reply]
Ugh-oh....
Looks like configuration issue.
Can you, please, open regedit, export `HKCU/Software/Alienfx_SDK` and `HKCU/Software/Alienfxgui` and attach it here?

**Reply by Rixtho** [reply]
Sure thing 👍🏾
[registries.zip](https://github.com/T-Troll/alienfx-tools/files/8856202/registries.zip)

**Reply by T-Troll** [reply]
Looks like configuration converter issue - all zones have no lights at all. And no grids...
Let me check what provide exception, as well as configuration converter.

**Reply by T-Troll** [reply]
Yes, it's a bug in configuration converter. Thank you to reveal it!
Please run this converter - [alienfx-conv.zip](https://github.com/T-Troll/alienfx-tools/files/8856610/alienfx-conv.zip) - it should fix and clean all. Say "yes" then it asks to override your settings, but better keep old for a while!

**Reply by Rixtho** [reply]
~~I downloaded the hotfix and tried running the new converter but `alienfx-gui` still doesn't open.~~ Got it working now 👍🏾

**Reply by T-Troll** [reply]
Great! Sorry for this, some ugly bugs quite possible after big changes.

**Comment by T-Troll** [comment]
[Hotfix](https://github.com/T-Troll/alienfx-tools/releases/tag/6.0.2.1) released - it fixes configuration converter issues and some other bugs.

**Reply by vinikmv** [reply]
Since I updated to the 6.0.2.1 version, the G-Mode is on by default.


If i close alien-fan, the G-Mode will fire up.
If i open alien-fan, I need to check the G-Mode or use the shortcut (fn+G key) to turn it off.

When I boot up, the G-Mode is turned on. Tried to run alienfx-conv but probably its not related to that.
In the 6.0.2 version, it worked correctly.

**Reply by T-Troll** [reply]
Yes. It's temporary, we are working on it - https://github.com/T-Troll/alienfx-tools/issues/123
